runtime/themes/jellybeans.toml

```diff
@@ -0,0 +1,137 @@
+#
+#    __       _ _       _                           
+#    \ \  ___| | |_   _| |__   ___  __ _ _ __  ___  
+#     \ \/ _ \ | | | | |  _ \ / _ \/ _  |  _ \/ __| 
+#  /\_/ /  __/ | | |_| | |_| |  __/ |_| | | | \__ \ 
+#  \___/ \___|_|_|\__  |____/ \___|\____|_| |_|___/ 
+#                 \___/                             
+#
+# Jellybeans
+# A take on the Jellybeans theme, please feel free to contribute! 
+#
+# Original author: @nanotech
+# Original repository: nanotech/jellybeans.vim
+# Contributors:
+# @cemalokten
+
+"attribute" = "green"
+"type" = "light_blue"
+"type.enum.variant" = "purple"
+"constructor" = "yellow" 
+"constant" = "dark_orange"
+
+"constant.builtin.boolean" = "yellow"
+"constant.character" = "yellow"
+"constant.characted.escape" = "red_error"
+"constant.numeric" = "dark_orange"
+"string" = "dark_green"
+"string.regexp" = "light_purple"
+"string.special" = { fg = "yellow", modifiers = ["underlined"] } 
+"comment" = "light_gray" 
+
+"variable" = "light_yellow" 
+"variable.builtin" = { fg = "dark_green", modifiers = ["underlined"] }
+"variable.parameter" = "yellow"
+"variable.other.member" = "light_purple"
+"label" = "yellow"
+"punctuation" = "mid_blue" 
+"keyword" = "mid_blue" 
+"keyword.control.exception" = "purple"
+"operator" = "light_purple"
+"function" = "yellow"
+"function.macro" = "green"
+"function.builtin" = "green"
+"function.special" = "green"
+"function.method" = "yellow"
+"tag" = "light_blue"
+"special" = "green"
+"namespace" = "light_purple"
+
+"markup.bold" = { fg = "white", modifiers = ["bold"] }
+"markup.italic" = { modifiers = ["italic"] }
```

Comment:
Similar to these lines, strikethough is possible. See https://github.com/helix-editor/helix/pull/5619